resource/js/vocab-counts.js

```diff
@@ -0,0 +1,54 @@
+const resourceCountsApp = Vue.createApp({
+  data () {
+    return {
+      concepts: {},
+      subTypes: {},
+      conceptGroups: {}
+    }
+  },
+  mounted () {
+    fetch('https://api.finto.fi/rest/v1/yso/vocabularyStatistics?lang=fi') /* What should this be? */
```

Comment:
Same as above, strip the `https://api.finto.fi/` URL prefix until we have a mechanism to define the REST base URL.